### PR TITLE
doc: update readme with deprecated notice

### PR DESCRIPTION
**Pull request by camilamacedo86**

### CODE REVIEW

**Review by varshaprasad96 — APPROVED** [review]
Minor change and lgtm from my end.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
+# [DEPRECATED] Operator SDK Samples
+
+This repository has been deprecated. The sample operator projects have been moved to `testdata/` in [Operator SDK][https://github.com/operator-framework/operator-sdk/] repository. For more info see the [enhancement proposal][https://github.com/operator-framework/enhancements/blob/master/enhancements/samples-generation.md] for automatic sample generation using Go.
```
Comment (jberkhahn) [inline README.md:3]: Need to fix these links. The second set of []s should be ()s.

Comment (jmrodri) [reply README.md:3]: Agreed. If you want to use square brackets you have to use an alias at the bottom of the document. 

```
[link text][link-alias]
...
[link-alias](actual/url)
```

OR

Do what @jberkhahn said and just change the second set of square brackets to parentheses:
```
[enhancement proposal](https://github.com/operator-framework/enhancements/blob/master/enhancements/samples-generation.md)
```
Which will render like this:
[enhancement proposal](https://github.com/operator-framework/enhancements/blob/master/enhancements/samples-generation.md)

**Review by jmrodri — CHANGES_REQUESTED** [review]
Fix the URL.

File: README.md (hunk 1 above)
Comment (camilamacedo86) [inline README.md:3]:
```suggestion
This repository has been deprecated. The sample operator projects have been moved to `testdata/` in [Operator SDK](https://github.com/operator-framework/operator-sdk/) repository. For more info see the [enhancement proposal](https://github.com/operator-framework/enhancements/blob/master/enhancements/samples-generation.md) for automatic sample generation using Go.  
```

Comment (camilamacedo86) [reply README.md]: ops. Typo issue.. Tks.

**Pull request merged**